### ISSUE TITLE
About the train of 512px model

### ISSUE THREAD

**Issue by FrankyCao** [lychenyoko/content-aware-gan-compression#9]
Thanks for open source. I try to train a 512px model after purne(ratio is 0.5)，but in the first 20000 iter, the fid score was very high（110+）, And is this normal.

**Comment by lychenyoko** [comment]
It doesn't seem normal to me. It would be good to check whether your FID is measured correctly and your generated images make sense visually. Otherwise, you may try lower the learning rate.

**Comment by FrankyCao** [comment]
Thank you for your reply.

PS:Dose you provide a compressed pretrained model of 512px？


I use this configuration to train：
training_iters = 990001
batch_size = 4
init_lr = 0.002
discriminator_r1 = 10
generator_path_reg_weight = 2
path_reg_batch_shrink = 2
g_reg_freq = 4
d_reg_freq = 16
noise_mixing = 0.9
val_sample_num = 25
val_sample_freq = 5000
model_save_freq = 10000
fid_n_sample = 5000
fid_batch = 32
kd_l1_lambda = 3
kd_lpips_lambda = 3
kd_mode = KNOWLEDGE_DISTILLATION_MODE[0]
content_aware_KD = True

**Comment by lychenyoko** [comment]
These results don't look good to me. Maybe you need to check the synthesis images from your teacher model and/or lower your learning rate by 10. 

No, I don't have checkpoint at 512px.

**Comment by FrankyCao** [comment]
thank you，I will try it again

**Comment by FrankyCao** [comment]
Hi，I check the teacher model and I lower learning rate and try again，but is not useful.
I think if the batchsize=4 is too samll？

**Comment by lychenyoko** [comment]
Yeah, that could be the issue. I used batch size of 16 to train both 256px and 1024px models. So it might be good to try 16 for your case.
